Source/Dafny/Verifier/Translator.ClassMembers.cs

```diff
@@ -1476,6 +1481,7 @@ public partial class Translator {
 
       var name = MethodName(m, kind);
       var proc = new Boogie.Procedure(m.tok, name, new List<Boogie.TypeVariable>(), inParams, outParams, req, mod, ens, etran.TrAttributes(m.Attributes, null));
+      AddVerboseName(proc, m.FullDafnyName, kind);
```

Comment:
You couldn't refactor this one?

Reply:
The benefit in this case (for `Procedure` instead of `Implementation`) is less clear, because there are only two instances of it and they differ in many dimensions.

However, reading this made me realize that there were some instances of the previous refactoring that I missed. I'll include those, too.